slave/buildslave/runprocess.py

```diff
@@ -317,7 +320,7 @@ def to_str(cmd):
         if not os.path.exists(workdir):
             os.makedirs(workdir)
         if environ:
-            for key, v in environ.iteritems():
+            for key, v in iteritems(environ):
```

Comment:
We don't. but I like having the code do one thing one way. since ```iteritems(environ)``` is the syntax used in other places, I also did it here